scania-digital-design-system/tegel
# feat(css): update from rem to px on utility classes

**Describe pull-request**  
From support request they are having trouble convering from sdds to tds when adding headline classes since the sdds html font size is different from tegel ones. We decided to change all rems to px to help them update.

**Solving issue**  
Fixes: [CDEP-2992](https://tegel.atlassian.net/browse/CDEP-2992)

**How to test**  
1. Go to storybook and typography, select headline
2. Make sure that the headlines have px instead of rem when inspecting the element and that its correct and aligned with design.

**Suggested test steps**

**Screenshots**  
<img width="1275" alt="Screenshot 2023-12-05 at 16 38 06" src="https://github.com/scania-digital-design-system/tegel/assets/605500/5db151ed-7c64-4d12-a919-36e78efb0600">


[CDEP-2992]: https://tegel.atlassian.net/browse/CDEP-2992?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
